ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -846,46 +846,31 @@ spec:
         - name: GITHUB_TOKEN_SECRET_KEY
           value: "$(params.github_token_secret_key)"
 
-    # Set the github status to success if the pipeline passed
-    - name: set-github-status-success
+    # Add the connect page URL in github PR comment to view the test results/logs
+    - name: github-add-connect-url-comment
       when:
-        - input: "$(tasks.status)"
+        - input: "$(tasks.certification-project-check.status)"
           operator: in
-          values: ["Succeeded", "Completed"]
+          values:
+            - Succeeded
```

Comment:
I don't think we need this `when` condition. Tekton should be smart enough to notice the dependency on `certification-project-check` and skip this task if the other didn't succeed.

Reply:
Yes, it seems no use of when condition here. I will remove it.